README.md

```diff
@@ -199,6 +199,33 @@ This area is enabled by default and includes the entities that aren't linked to
 The area can be configured like any other area as described above.
 To exclude this area from the dashboard and views, set its property `hidden` to `true`.
 
+#### Settings for all areas
+
+Use `_`to set options for all areas. Example:
+```yaml
+strategy:
+  type: custom:mushroom-strategy
+  options:
+    areas:
+      _:
+        layout: vertical
+```
+
+##### Access to area ID
+
+If you want access to the area ID, you can use the value `"{{area_id}}"`. Example:
```

Comment:
No need for this kind of evaluation if you create a new class for an HA Area card.

Reply:
Done